types/htmlhint/htmlparser.d.ts

```diff
@@ -0,0 +1,41 @@
+export interface Attr {
+    name: string;
+    value: string;
+    quote: string;
+    index: number;
+    raw: string;
+}
+
+export interface Block {
+    tagName: string;
+    attrs: Attr[];
+    type: string;
+    raw: string;
+    pos: number;
+    line: number;
+    col: number;
+    content: string;
+    long: boolean;
+    close: string;
+    lastEvent?: Partial<Block>;
+}
+
+export type Listener = (event: Block) => void;
+
+export interface Pos {
+    line: number;
+    col: number;
+}
```

Comment:
this does not exists in original, those are just adhoc, inline return types (single usage):
```ts
fixPos(event: Block, index: number): {
        line: number;
        col: number;
    };
 `